src/components/Creators/Testimonials.vue

```diff
@@ -0,0 +1,293 @@
+                    />
+                  </div>
+                  <div class="ml-4 lg:ml-0">
+                    <div
+                      class="
+                        text-xl
+                        font-medium
+                        text-gray-900
+                        dark:text-gray-100
+                        underline
+                        inline-flex
+                        items-center
+                        justify-around
+                      "
+                    >
+                      {{ currentItem.name }}
+                    </div>
+                  </div>
+                </div>
+              </a>
+            </footer>
+            <div class="py-8 flex justify-between items-center">
+              <p
+                class="
+                  pl-0
+                  px-4
+                  lg:px-8
+                  lg:pl-0
+                  underline
+                  font-bold
+                  mt-6
+                  font-mono
+                  justify-self-end
+                  cursor-pointer
+                "
+                v-on:click="setCurrentItem(prevItem())"
+              >
+                &lt; Previous
+              </p>
+              <p
+                class="
+                  pr-0
+                  px-4
+                  lg:px-8
+                  lg:pr-0
+                  underline
+                  font-bold
+                  mt-6
+                  font-mono
+                  justify-self-end
+                  cursor-pointer
+                "
+                v-on:click="setCurrentItem(nextItem())"
+              >
+                Next &gt;
+              </p>
+            </div>
+          </blockquote>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  name: "Testimonial",
+  data() {
+    return {
+      items: [
+        {
+          name: "Sindre",
+          image: "https://github.com/sindresorhus.png",
+          description: `"I like how the Dev Protocol is improving open source sustainability in an innovative way"`,
+          link: "https://twitter.com/sindresorhus",
+        },
+        {
+          name: "μ-zomia",
+          image: "/assets/m-zomia.png",
+          description: `"DevProtocol is born out of the love of people supporting people. Once you understand that, there's no reason not to be part of the team."`,
+          link: "https://twitter.com/zomia12",
+        },
+        {
+          name: "HiDE",
+          image: "/assets/hide.png",
+          description: `"DEV protocol is a sustainable source of funding for the community ecosystem that uses our services to operate.
+It is a core feature of the HiÐΞ protocol and is a very important financial resource. As OSS creators, we feel free to use some DEV tokens for development and redistribute them in our services. I will continue to do my best for the further development of the DEV community that you have staked out for us."`,
```

Comment:
```suggestion
It is a core feature of the HiÐΞ protocol and is a very important financial resource. As OSS creators, we feel free to use some DEV tokens for development and redistribute them to our services. I will continue to do my best for the further development of the DEV community that you have staked out for us."`,
```

Reply:
I will contact HiDE about this, this is a testimonial, we can't change anything unless they allow it.

Reply:
We can't change someone's testimonial, right? 🙂

Reply:
Yes.